carbon-design-system/carbon
# fix(select): remove label on svg chevron

Closes #3451
Closes #3449
Closes #4222

#### Changelog

**New**

**Changed**

- Remove `aria-label` and `role="img"` from icon
- Deprecate `iconDescription` prop
- Remove tests for `iconDescription`

**Removed**

#### Testing / Reviewing

- Verify no DAP violations exist for the icon (which had `role="img"`):
- Verify fixes closed issues on JAWS and VoiceOver
